src/kernel/locking_heap.c

```diff
@@ -1,10 +1,13 @@
 #include <kernel.h>
+#include <management.h>
 
 typedef struct heaplock {
     struct heap h;
     struct spinlock lock;
     heap parent;
     heap meta;
+    tuple mgmt;
+    tuple parent_mgmt;
```

Comment:
parent_mgmt doesn't get initialized, although it doesn't seem like it would be accessed before initialization.

Reply:
Thanks for catching these. I pushed a fix.